core/domain/playthrough_issue_registry_test.py

```diff
@@ -18,14 +18,59 @@
 
 from __future__ import annotations
 
+from core import utils
 from core.domain import playthrough_issue_registry
 from core.tests import test_utils
+from extensions.issues.CyclicStateTransitions import CyclicStateTransitions
+from extensions.issues.EarlyQuit import EarlyQuit
+from extensions.issues.MultipleIncorrectSubmissions import (
+    MultipleIncorrectSubmissions)
 
 
 class IssueRegistryUnitTests(test_utils.GenericTestBase):
     """Test for the issue registry."""
 
-    def test_issue_registry(self):
+    def setUp(self):
+        super(IssueRegistryUnitTests, self).setUp()
+        self.issues_dict = {
+            'EarlyQuit': (
+                EarlyQuit.EarlyQuit),
+            'CyclicStateTransitions': (
+                CyclicStateTransitions.CyclicStateTransitions),
+            'MultipleIncorrectSubmissions': (
+                MultipleIncorrectSubmissions.MultipleIncorrectSubmissions)
+            }
+        self.invalid_issue_type = 'InvalidIssueType'
+
+# Adding test execution order with 01, 02, 03 because playthrough_issue_registry
```

Comment:
The indentation here is weird and the approach also doesn't seem great; it should be possible to run tests in any order. Could you please figure out a way to refactor these tests so that the above condition holds? You can do setups and teardowns for individual tests within the test functions, so that you have the right conditions holding at the right times.

Reply:
Sure !! I will try to find some workarounds to fix this issue 😃